README.md

```diff
@@ -1,19 +1,113 @@
+  - `/jira instance` - Manage connected Jira instances. Must have System Admin role in Mattermost.
 
-Run `make vendor` to install dependencies, then develop like any other Go project: Hack away and use `go test`, `go build`, etc.
+The above are supported for Jira Core and Jira Software, both for Cloud and Server platforms.
 
-If you want to create a fully bundled plugin that will run on a local server, you can use `make mattermost-jira-plugin.tar.gz`.
+## 2. Jira Plugin v2.0 Roadmap
 
-## Releasing
+Below is a full list of features planned for Jira plugin v2.0.
 
-To make a release, update the version number in plugin.yaml, and create a release via the GitHub interface. Travis will upload the distributables for you.
+We are looking for help from our community to complete the development of v2.0 of the Mattermost Jira plugin. If you're interested, review [open Help Wanted issues](https://github.com/mattermost/mattermost-plugin-jira/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) to get started.
+
+- Notifications for 10+ issue events from Jira to Mattermost with full JQL support, using webhooks. This includes notifications for the following events: issue created; issue transitioned to “Reopened”, “In Progress”, "Submitted" or “Resolved”; issue deleted or closed; comments created, updated or deleted; assignee, title, description, priority, sprint or rank updated; attachments or labels added; attachments or labels removed
+- Create Jira issues via Mattermost UI (Desktop App and browser only)
+- Attach Mattermost messages to Jira issues via Mattermost UI (Desktop App and browser only)
+- Preview Jira issues in Mattermost when a ticket is referenced
+- Send direct messages for Jira at-mentions and issue assignments
+- Slash commands for
+  - `/jira connect` - Connect to a Jira project and subscribe to events.
+  - `/jira disconnect` - Disconnect from a Jira project and subscribe from events.
+  - `/jira subscribe` - Subscribe a Mattermost channel to receive notifications for above issue updates in a Jira project.
+  - `/jira instance` - Manage connected Jira instances. Must have System Admin role in Mattermost.
+  - `/jira create [description]` - Create a Jira ticket.
+  - `/jira assign <issue-key> <assignee>` - Assign a Jira issue specified by <issue-key>. <assignee> must be a member of the Jira project.
+  - `/jira transition <issue-key> <state>` - Transition a Jira issue specified by <issue-key>. <state> must be a valid Jira state such as "Done".
+  - `/jira settings preview [on/off]` - Set whether previews of Jira issues are shown.
+  - `/jira settings notifications [on/off]` - Set whether Direct Message notifications are sent for assignments and comments in assigned issues.
+  - `/jira view <issue-key>` - View a Jira issue specified by <issue-key>.  
+
+## 3. Installation
+
+### 3.1 Jira Cloud (Core + Software)
+
+#### 3.1.1 Mattermost
+
+1. Download this binary: // XXX
```

Comment:
Need to add link to binary here

Reply:
https://s3.amazonaws.com/mattermost-public-plugins-kubernetes/jira-test-2.0.0.tar.gz should work